_redirects

```diff
@@ -1,2 +1,2 @@
-/cortex/*   https://reference80.epdemos.com/cortex/:splat      200
+/cortex/*   https://dev1-cortex-gw.ep-dev.aws.punchout2go-test.com/cortex/:splat          200
```

Comment:
We'll need to update this before we merge

Reply:
WIP